packages/server/src/sdk/app/applications/general.ts

```diff
@@ -0,0 +1,19 @@
+const URL_REGEX_SLASH = /\/|\\/g
```

Comment:
The `general` term stands out a bit, it's not a name we've used a lot. `utils` feels more familiar. Or this file could become the beginning on the applications sdk itself, i.e. the main file we use for create, read, update etc, when it then would be named more like `/sdk/applications/applications.ts`

Reply:
or a few other suggestions

`/sdk/applications/`:
- `appService.ts`
- `appSdk.ts` 
- `service.ts`
- `sdk.ts` 

we should probably agree something and get a doc going on confluence for explaining project structure and naming conventions for new hires